.circleci/config.yml

```diff
@@ -118,6 +118,28 @@ jobs:
             - target/release/deps
           key: v4-cargo-cache-darwin-{{ arch }}-{{ checksum "Cargo.lock" }}
 
+  test-rust-nightly:
+    docker:
+      - image: circleci/rust:latest
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
+      - run: sudo apt-get install -y cmake
+      - run: rustup default nightly
+      - run: make test
+      - run: rustup component add rustfmt
+      - run: make lint
```

Comment:
Maybe nightly lint will differ from stable one.
I think we should remove this one (to only test for lint once)

Reply:
Removed the lint. I agree.